cmd/integration/integration.go

```diff
@@ -807,7 +893,7 @@ func runServiceTest(client *client.Client) {
 	if err != nil {
 		glog.Fatalf("Failed to create pod: %v, %v", pod, err)
 	}
-	if err := wait.Poll(time.Second, time.Second*20, podExists(client, pod.Namespace, pod.Name)); err != nil {
+	if err := wait.Poll(time.Second, time.Second*30, podExists(client, pod.Namespace, pod.Name)); err != nil {
```

Comment:
because it was failing intermittently at 20 when I was running integration tests on a stressed laptop